src/pegin/components/create/BtcInput.vue

```diff
@@ -62,20 +62,24 @@ import {
   isBTCAmountValidRegex,
   toWeiBigIntString,
   generateMockRSKAddress,
-  getBridgeLockingCap,
   bigIntToUserFormattedWei,
 } from '@/common/utils';
 
 export default defineComponent({
   name: 'BtcInput',
+  props: {
+    flyoverAvailable: {
+      type: Boolean,
+      default: false,
+    },
+  },
   setup(props, context) {
     const environmentContext = EnvironmentContextProviderService.getEnvironmentContext();
     const peginConfiguration = useStateAttribute<PeginConfiguration>('pegInTx', 'peginConfiguration');
     const liquidityProviders = useStateAttribute<LiquidityProvider2WP[]>('flyoverPegin', 'liquidityProviders');
-    const flyoverEstimateGasFee = useAction('flyoverPegin', constants.FLYOVER_PEGIN_ESTIMATE_QUOTE_FEE);
+    const flyoverEstimateGasFee = useAction('flyoverPegin', constants.FLYOVER_PEGIN_ESTIMATE_QUOTE_MAX_FEE);
```

Comment:
Nit: to avoid the `as WeiBig` assertion below.
```suggestion
    const flyoverEstimateGasFee = useAction<Promise<WeiBig>>('flyoverPegin', constants.FLYOVER_PEGIN_ESTIMATE_QUOTE_MAX_FEE);
```